include/attribute.h

```diff
@@ -0,0 +1,282 @@
+#ifndef GUARD_ATTRIBUTE_H
+#define GUARD_ATTRIBUTE_H
+
+#include "global.h"
+
+struct RoleAttrib
+{
+    u8 filler0[0x78];
+};
+
+struct BagWeapon
+{
+    u8 filler0; //something related to reforge
+    u8 weaponType;
+    u8 weaponSpecialEffect;
+    u8 weaponSkill;
+    u8 firstEnhanceItem; //null = 0xff
+    u8 secondEnhanceItem;
+    u8 thirdEnhanceItem;
+    u8 weaponTec;
+    u16 weaponAtk;
+    u16 weaponDef;
+    u16 weaponAgl;
+    u16 weaponInitAtk; //Atk when not enhanced or boosted
+    u16 weaponInitDef;
+    u16 weaponInitAgl;
+    u16 weaponCurrentDur;
+    u16 weaponMaxDur;
+    u16 weaponInitDur;
+    u16 isGoodWeapon;
+}; /* size = 0x1C */
+
+struct SaveBlock1
+{
+    u8 playerGender; //00 = male
+    u8 playerLevel; 
+    u8 bladeForgeLevel; //00-09
+    u8 axeForgeLevel;
+    u8 spearForgeLevel;
+    u8 boxGloveForgeLevel;
+    u8 bowForgeLevel;
+    u16 filler8[0xE]; //Something related to player name
+    u16 playerMaxHp; //max = 0x270f,but more is ok
+    u16 playerCurrentHp;
+    u16 equippedOneAtk; //max = 0x3e7
+    u16 equippedTwoAtk;
+    u16 equippedThreeAtk;
+    u16 notEquippedAtk;
+    u16 filler30; //TODO:maybe this is a substruct align
+    u16 equippedOneDef;
+    u16 equippedTwoDef;
+    u16 equippedThreeDef;
+    u16 notEquippedDef; 
+    u16 filler3A;
+    u16 equippedOneAgl;
+    u16 equippedTwoAgl;
+    u16 equippedThreeAgl;
+    u16 notEquippedAgl; 
+    u16 filler44;
+    u8 equipOneIndex; //max = 0x1D, as one can take up to 30 wepons in the bag
+    u8 equipTwoIndex;
+    u8 equipThreeIndex;
+    u8 equipItemIndex;
+    u16 filler4A;
+    u8 filler4C[8];
+    u16 filler54;
+    u16 bonusStats; //synced with 020036BA, void?
+    u32 filler58;
+    u32 playerExp; //max = 00BD95FC
+    u32 filler60;
+    u8 guardimalType; //00-04
+    u8 guardimalLevel;
+    u8 filler66[0xE]; //Something related to guardimal name
+    u16 guardimalHp;
+    u16 filler76;
+    u16 guardimalMag;
+    u16 guardimalAtk;
+    u16 guardimalDef;
+    u16 guardimalAgl;
+    u32 filler80;
+    u32 guardimalExp;
+    u32 guardimalNextExp;
+    u8 battleItemOne;
+    u8 battleItemTwo;
+    u8 battleItemThree;
+    u8 battleItemFour;
+    u8 filler90[0x54];
+    struct BagWeapon bagWeapon[0x1E]; 
+    u8 filler42C[0x1E];
+    u8 numWoundBand;
+    u8 numJvhuaCha; // TODO:rename this
```

Comment:
I'm really confused because sometimes you have a space after `//` but sometimes not.